app/Controllers/BaseController.php

```diff
@@ -3,44 +3,28 @@
 namespace App\Controllers;
 
 use CodeIgniter\Controller;
-use CodeIgniter\HTTP\CLIRequest;
-use CodeIgniter\HTTP\IncomingRequest;
 use CodeIgniter\HTTP\RequestInterface;
 use CodeIgniter\HTTP\ResponseInterface;
 use Psr\Log\LoggerInterface;
 
 /**
- * Class BaseController
- *
  * BaseController provides a convenient place for loading components
  * and performing functions that are needed by all your controllers.
+ *
  * Extend this class in any new controllers:
+ * ```
  *     class Home extends BaseController
+ * ```
  *
- * For security be sure to declare any new methods as protected or private.
+ * For security, be sure to declare any new methods as protected or private.
  */
 abstract class BaseController extends Controller
 {
-    /**
-     * Instance of the main Request object.
-     *
-     * @var CLIRequest|IncomingRequest
-     */
-    protected $request;
-
-    /**
-     * An array of helpers to be loaded automatically upon
-     * class instantiation. These helpers will be available
-     * to all other controllers that extend BaseController.
-     *
-     * @var list<string>
-     */
-    protected $helpers = [];
```

Comment:
this part probably better just commented so user know that it can be enabled and added with list of helpers:

```php
//    protected $helpers = ['array'];
```

Reply:
I agree. People may not realize this option exists, since they normally won't dig into the source of the Controller class.

Reply:
Added inside the `initController`.